recipes/tensorstore/patches/pybind11-linkopts.patch

```diff
@@ -0,0 +1,13 @@
+--- python/tensorstore/BUILD	2022-07-18 11:20:35.000000000 +0200
++++ python/tensorstore/BUILD	2022-07-18 11:23:52.000000000 +0200
+@@ -27,6 +27,10 @@
+     name = "_tensorstore",
+     srcs = ["tensorstore.cc"],
+     imports = [".."],
++    linkopts = select({
```

Comment:
Do you know why this is needed given that we don't seem to need it for our own package builds?

Reply:
I don't think this is needed. We have disabled macos for now. I'll ping you in another PR when I enable macos so that I can get some feedback.